imagebuildah/build.go

```diff
@@ -899,11 +899,15 @@ func (s *StageExecutor) Execute(ctx context.Context, stage imagebuilder.Stage) e
 		}
 
 		if i < len(children)-1 {
-			s.output = ""
+			commitName = ""
 		} else {
-			s.output = commitName
+			commitName = s.output
 		}
 
+		// TODO: this makes the tests happy, but it shouldn't be
+		// necessary unless this is the final stage.
+		commitName = s.executor.output
```

Comment:
This overwrites the command just above?

Reply:
Yes.  For the moment, not doing this causes tests to stop passing.
There's a bigger change to the execute loop coming up that'll require updating the tests, and this will be resolved as part of that, but this PR is intended to be a smaller change that doesn't impact tests, so in the interests of that, for now I'm just marking it as a to-do item.